README.md

```diff
@@ -291,65 +290,70 @@ and it is far easier to work with them when you have deeper understanding yourse
-    For our purposes, it’s best just to think of it as the young period of a fast-growing
-    company.
-  - 🚧 Groundwork:
-    What are good stats on how many people work in startups vs established companies?
-- Companies in the US include sole proprietorships, partnerships, limited liability
-  companies (LLCs), S corporations, and C corporations.
-  - 🔸 This Guide does not cover equity compensation in LLCs, as they are rarely used as the
-    choice of entity for technology startups, and equity compensation in LLCs is dramatically
-    different from equity compensation in corporations.
-    So we’ll focus on the one type of company most readers of this Guide will be concerned
-    with: The C corporation.
-- 🄳 A [**C corporation (or C corp)**](https://en.wikipedia.org/wiki/C_corporation) is a type of
-  business entity in the United States.
+  are common, including sole proprietorships, partnerships, limited liability companies
+  (LLCs), S corporations, and C corporations.
+  Several of these kinds are called **corporations** and they are formed, or
+  [**incorporated**](https://en.wikipedia.org/wiki/Incorporation_(business)), under the laws of a
+  specific state.
+- 🔸 This Guide [does not cover](#scope) equity compensation in LLCs, S corporations,
+  partnerships, or sole proprietorships.
+  Equity compensation in each of these kinds of business entities is significantly different
+  than in C corporations.
+- 🄳 A [**C corporation (or C corp)**](https://en.wikipedia.org/wiki/C_corporation) is one type of
+  corporation United States.
   Specifically, it’s a kind of
-  [**stock corporation**](https://en.wikipedia.org/wiki/Joint-stock_company) with certain federal
-  tax treatment.
-  Most large companies you’ve heard of, as well as startups that aspire to be large, are C
-  corporations. In practice, for a
+  [**stock corporation (or joint-stock company)**](https://en.wikipedia.org/wiki/Joint-stock_company))
+  with certain federal tax treatment.
+  Many of the high-profile companies you hear about are C corporations.
+- In practice, for a
   [few](https://www.nytimes.com/2012/07/01/business/how-delaware-thrives-as-a-corporate-tax-haven.html)
   [reasons](https://www.quora.com/Why-do-most-technology-startups-incorporate-in-Delaware),
   these companies are usually formed in Delaware, so legalities of all this are defined in
   Delaware law.
-- Loosely, one way to think about all this is that a company is a set of contracts,
-  negotiated over time between people and enforced by the government, that align interests
-  of a group of people in creating things customers are willing to pay for.
-  A key part of these contacts is precisely tracking ownership.
-  Historically, this has been done a few ways, depending on the size and nature of the
-  company, but possibly the most effective way when a lot of people are involved, is via
-  stock, which we get into next.
+
+Some of what we discuss is specific to early stage companies.
```

Comment:
This needs an explanation. We do cover why companies use equity comp in the intro, but if not reading top to bottom this comes out of nowhere. Why is some of what we discuss specific to early stage companies? I believe adding Startups info to a section on Public and Private Companies (see PR #114) offers more clarity from a structural standpoint and more room to explain why those companies (inc startups) use equity comp.

Reply:
OK, for now will adjust language.